python/tank/descriptor/descriptor.py

```diff
@@ -358,7 +358,7 @@ def support_url(self):
         meta = self._get_manifest()
         support_url = meta.get("support_url")
         if support_url is None:
-            support_url = "https://support.shotgunsoftware.com"
+            support_url = "https://developer.shotgridsoftware.com"
```

Comment:
This one is tricky. That variable is used in different places and it is mostly mean to contact support. I think we might need to use this URL: https://help.autodesk.com/view/SGSUB/ENU/

What do you think?